capstone/client/src/components/ListPage.js

```diff
@@ -129,12 +119,21 @@ class ListPage extends Component {
   }
 
   selectList = (event) => {
-    var index = event.target.name;
-    if(event.target.className === "MuiButton-label"){
-      index = event.target.parentElement.name;
+    const index = event.target.value;
+    if (index === undefined) {
+      return;
+    }
+    if (index === -1) {
+      this.setState({
+        listId: -1,
+        listIndex: -1,
```

Comment:
We're using too many arbitrary values. index can be undefined or -1, which is very error-prone. Can you just set them to null or undefined?

Reply:
I am using -1 because the API endpoint uses -1 for creating a new list. The undefined is for when a user click somewhere else on the screen instead of clicking one of the buttons from the menu.